packages/web-app-files/src/components/SideBar/Shares/Collaborators/NewCollaborator.vue

```diff
@@ -119,15 +123,16 @@ export default {
 
     $_isValid() {
       return this.selectedCollaborators.length > 0
+    },
+
+    minSearchLength() {
+      return parseInt(this.user.capabilities.files_sharing.search_min_length, 10)
     }
   },
   mounted() {
-    // Ensure default role is not undefined
-    this.$nextTick(() => {
-      this.$refs.ocSharingAutocomplete.focus()
-    })
+    this.focusInviteInput()
 
-    this.$_onAutocompleteInput = debounce(this.$_onAutocompleteInput, 1000)
+    this.fetchRecipients = debounce(this.fetchRecipients, 1000)
```

Comment:
1 second could be a too long debounce time, i tested it with 500ms and it felt less "tough"